src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -41,6 +43,8 @@
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\amd64\git2-69db893.dll" target="lib\NativeBinaries\amd64" />
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\x86\git2-69db893.dll" target="lib\NativeBinaries\x86" />
     <file src="Microsoft.DotNet.Build.Tasks\AssemblyInfoPartial.*" target="lib" />
+    <file src="GenFacades\GenFacades.dll" target="lib" />
```

Comment:
If we plan to use this with CoreRun, we can probably stop building it with CoreConsole and having it rename the extension from exe -> dll. There's a post-step in the publishexe.targets file that renames the file if it detects CoreConsole.exe next to the project output.